frontend/cypress_system_tests.json

```diff
@@ -0,0 +1,13 @@
+{
+  "baseUrl": "http://localhost:3000",
+  "viewportWidth": 1920,
+  "viewportHeight": 1080,
+  "component" : {},
+  "integrationFolder": "cypress/system",
+  "pluginsFile": false,
+  "env": {
+    "APP_KEY": "f9c23feb7e5e70217a40516490d6ed3572c863f27716e183dbbc37d7d10dfb928434acea06a05637bfb7f294652336cb08789866e57ce41f81856bf15bd3c8f0dc646f32ab7fb04f6b437850941d0f9beefd3aa95097f06c6a5cd6e567594a1fc023d529b217ac33cb266124de2647a6bd0666acef34ca8eba4332e56f64a13a",
+    "APP_ENDPOINT": "http://localhost:81/v1",
```

Comment:
is the port 81 the correct one?

Reply:
81 was just a default value. As a usually have something else running on port 80, I need to use another port.